### ISSUE TITLE
update needed to make_instmap_weights change for sherpa i/f

### ISSUE THREAD

**Issue by kglotfelty** [cxcsds/ciao-contrib#258]
I'm seeing a failure in tests

    # make_instmap_weights (06 June 2019): ERROR 'Data1DInt' object has no attribute '_get_mask'

I'm assuming this is related to the object refactoring in sherpa 4.11.1

**Comment by DougBurke** [comment]
Thanks. I wonder if I call` _get_mask` directly (which is not ideal) or if this is a more-subtle issue.

**Comment by kglotfelty** [comment]
looks like it's in `sherpa_contrib`


```bash
% make_instmap_weights out=/pool1/kjg/foo model='const1d.c' paramvals='c.c0=0.001' \
  emin=0.5 emax=8.0 ewidth=0.1 clob+ mode=h --debug
 Solar Abundance Vector set to angr:  Anders E. & Grevesse N. Geochimica et Cosmochimica Acta 53, 197 (1989)
 Cross Section Table set to bcmc:  Balucinska-Church and McCammon, 1998
Model used: 
const1d.c
   Param        Type          Value          Min          Max      Units
   -----        ----          -----          ---          ---      -----
   c.c0         thawed        0.001 -3.40282e+38  3.40282e+38           
# make_instmap_weights (06 June 2019): ERROR 'Data1DInt' object has no attribute '_get_mask'


##### Traceback:

Traceback (most recent call last):
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/lib/python3.5/site-packages/ciao_contrib/logger_wrapper.py", line 531, in new_fn
    return fn(*args, **kw)
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/bin/make_instmap_weights", line 177, in make_instmap_weights
    save_instmap_weights(params["outfile"], params["clobber"])
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/lib/python3.5/site-packages/sherpa_contrib/utils.py", line 717, in save_instmap_weights
    wgts = get_instmap_weights(user["id"], fluxtype=user["fluxtype"])
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/lib/python3.5/site-packages/sherpa_contrib/utils.py", line 600, in get_instmap_weights
    return InstMapWeights1DInt(id, fluxtype=fluxtype)
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/lib/python3.5/site-packages/sherpa_contrib/utils.py", line 142, in __init__
    self._apply_mask(d)
  File "/export/CIAOT/ciaot_install/20190729/ciao-4.12/contrib/lib/python3.5/site-packages/sherpa_contrib/utils.py", line 192, in _apply_mask
    m = data._get_mask()
AttributeError: 'Data1DInt' object has no attribute '_get_mask'
```

**Comment by DougBurke** [comment]
This should be fixed in #269 (which hasn't been merged yet).
